scripts/migrate_to_turn/update_turn_contacts_queue.py

```diff
@@ -0,0 +1,58 @@
+import asyncio
+import csv
+import os
+import sys
+from urllib.parse import urljoin
+
+import aiohttp
+
+from scripts.migrate_to_rapidpro.retry_requests import request
+
+WORKER_COUNT = 3
+
+TURN_URL = "https://whatsapp-praekelt-cloud.turn.io"
+
+
+async def update_turn_contact_details(session, row, target):
+    wa_id = row.pop("wa_id")
+
+    url = urljoin(TURN_URL, f"/v1/contacts/{wa_id}/profile")
+    headers = {
+        "Authorization": f"Bearer {os.environ['TURN_TOKEN']}",
+        "content-type": "application/json",
+        "Accept": "application/vnd.v1+json",
+    }
+    await request(session, url, "PATCH", headers, row, target)
+
+
+async def worker(name, queue):
+    while True:
+        session, row, target = await queue.get()
+        await update_turn_contact_details(session, row, target)
+        queue.task_done()
+
+
+async def main(filename, target):
+    queue = asyncio.Queue()
```

Comment:
We should add a max size to the queue (probably the number of workers), so that it blocks while adding items to the queue, so that we don't read all the rows at once. Combined with `await queue.put(update)`, and starting the workers before we start adding the tasks.